### ISSUE TITLE
Error when deleting in batch

### ISSUE THREAD

**Issue by DEKHTIARJonathan** [viewflow/django-material#267]
Hello dear friends,

When I try to delete any object from the admin panel by **ticking the boxes**, I have the exact same error:

```
Django Version:	1.11
Exception Type:	ValueError
Exception Value:	Invalid literal for int() with base 10: '7" class="action-select'
```

If we go down to the **Request information** section, the following information is displayed:

![image](https://cloud.githubusercontent.com/assets/10923599/24980227/a70c15ac-1fd7-11e7-887c-3a12f0f1dc73.png)

There is clearly a bug with **_selected_action** it should be equal to '7' and not '7" class="[...]'

Do you confirm it is an issue with Django Material or should I look somewhere else ?

Best Regards,

Jonathan

**Comment by kmmbvnr** [comment]
yep, happens on django>1.11

**Comment by hairychris** [comment]
It's because the quotes are messed up, I should have a fix pushed in a few minutes:

````
<input class="action-select" name="_selected_action" id="_selected_action_1" type="checkbox" value="QzG4mWK&quot; class=&quot;action-select">
````

**Comment by hairychris** [comment]
It might have something to do with this line https://github.com/viewflow/django-material/blob/91c6f02d4e822ef1102a882c279852cd28d6e11d/material/admin/templatetags/material_admin.py#L327

**Comment by hairychris** [comment]
If you change the regular expression on this line https://github.com/viewflow/django-material/blob/91c6f02d4e822ef1102a882c279852cd28d6e11d/material/admin/templatetags/material_admin.py#L27

````
CL_VALUE_RE = re.compile('value="(.*)" class')
````

It works but it feels like a hack.

**Comment by DEKHTIARJonathan** [comment]
@kmmbvnr would it be possible to push a corrected version on PyPi  ?
Thanks a lot.

Jonathan

**Comment by kmmbvnr** [comment]
next open-source version release would be in a month or two.

**Comment by hairychris** [comment]
Apparently it's easier to poo on what I did than to implement and push a fix.

**Comment by hairychris** [comment]
DEKHTIAR you can add this to your Pip install for now if you want it to work:

https://github.com/hairychris/django-material/archive/2b3d70347cf29bcc02b06d3319f9617b626502c8.zip

**Comment by kmmbvnr** [comment]
@hairychris your fix is wrong, and probably don't work on other djangos.

**Comment by hairychris** [comment]
Probably or doesn't? There is a huge world of difference.

**Comment by hairychris** [comment]
The thing is if someone pip installs from my fork everything works, if they do it from your original package it does not. If you have a more pure fix I'm all ears. If you don't surely we are better off having something working non optimally than something not working at all?

**Comment by hairychris** [comment]
I'm not saying blindly pull it but seeing as I triaged the issue, found the line it was happening on and proposed a potential fix perhaps some collaboration rather than conflict would benefit us all.

**Comment by kmmbvnr** [comment]
Fix c77436153c77c30eaa4d67c32514b9125ac61e99

**Comment by DEKHTIARJonathan** [comment]
@kmmbvnr Could you re-open this issue ?
It is still a problem with the new release 1.0.0 ... 

Thanks a lot,

Jonathan
